dom.bs

```diff
@@ -2427,10 +2427,8 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
    its <a for=range>end offset</a> by <var>count</var>.
   </ol>
 
- <li>Let <var>nodes</var> be <var>node</var>'s
- <a>children</a> if <var>node</var> is
- a {{DocumentFragment}} <a>node</a>, and a
- list containing solely <var>node</var> otherwise.
+ <li><p>Let <var>nodes</var> be <var>node</var>'s <a>children</a>, if <var>node</var> is a
+ {{DocumentFragment}} <a>node</a>, « <var>node</var> » otherwise.
```

Comment:
This reads really wrong; I'm not happy with where we've landed if this is what happens. Commas just don't work that way. Semicolon could work, or using a conjunction (either "and" or "or").

Reply:
Same if otherwise comes first? I see now I put the wrong way around. I guess I can do some follow-up on Monday in that original thread on this.

Reply:
Ah I see. If otherwise comes first it isn't as viscerally wrong, but I still have a slight preference for a conjunction or semicolon.